src/test/kotlin/edu/carroll/polyominoes/service/account/RegisterServiceImplTest.kt

```diff
@@ -0,0 +1,289 @@
+package edu.carroll.polyominoes.service.account
+
+import jakarta.transaction.Transactional
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+
+@Transactional
+@SpringBootTest
+internal class RegisterServiceImplTest(
+        @Autowired private val registerServiceImpl: RegisterServiceImpl
```

Comment:
By convention, the annotation should be on a separate line from the declaration.